pedalboard/ExternalPlugin.h

```diff
@@ -193,6 +193,19 @@ template <typename ExternalPluginType> class ExternalPlugin : public Plugin {
           pathToSharedObjectFile.getFullPathName().toStdString() + "\"` to " +
           "see which dependencies might be missing.).");
 #else
+
+// On certain Macs, plugins will only load if installed in the appropriate path.
+#if JUCE_MAC
```

Comment:
Since this exists in a function you can use [if constexpr](https://en.cppreference.com/w/cpp/language/if) instead. Should be safer than relying on the C macro preprocessor.